### ISSUE TITLE
15 minute cooldown.... v 14

### ISSUE THREAD

**Issue by Logikov** [Jessecar96/SteamDesktopAuthenticator#947]
Good day. In version 1.0.10, when the number was linked to the account, if there was no SMS, then you could try a new phone number. Now you need to wait 15 minutes for a form with a phone number to appear. Can remove this internal cooldown?

**Comment by CraniumCS** [comment]
if you're still having the same issue maybe try this video, worked for me but yours might be different I am on windows 10 64 bit hope it helps! https://www.youtube.com/watch?v=1yBN5-QBHzM

**Comment by therepower** [comment]
> if you're still having the same issue maybe try this video, worked for me but yours might be different I am on windows 10 64 bit hope it helps! https://www.youtube.com/watch?v=1yBN5-QBHzM

@Jessecar96 posting scam links

**Comment by merelenten** [comment]
Yes, I confirm that the program either does not work or there is no function to cancel the number binding. Therefore, we have to wait for these 15 minutes when the steam cancels (resets) this process due to inactivity. This is very important, please fix it in the new version!

**Comment by merelenten** [comment]
In order not to wait 15 minutes to change the number, you need to go through the browser to the number binding page on steam.  Insert any phone number, click OK and then click cancel. After that, you can run sda again and make a binding with a new number. This is like a temporary solution , since sda there  is no function to cancel the number in sa yet , or it does not work .
Note: a confirmation email will appear in the mailbox for the phone number whose binding was initiated from the browser, you just need to ignore it)
